backend/Dockerfile

```diff
@@ -1,15 +1,14 @@
-FROM python:3.11.1-alpine3.16
+FROM python:3.11-alpine3.16
 
 # Set work dir for relative paths
-RUN mkdir -p /app
```

Comment:
I believe the `mkdir` here might not be necessary as `WORKDIR` below should create the directory if it does not yet exist.

Reply:
Didn't know that! Thanks for explaining :)